src/FirebaseArduino.h

```diff
@@ -24,18 +24,97 @@
 #define FIREBASE_JSONBUFFER_SIZE 200
 #endif // FIREBASE_JSONBUFFER_SIZE
 
+/**
+ * Gateway class for Arduino clients to interact with the Firebase backend.
```

Comment:
Maybe we should also precise there is the default `Firebase` instance and give an example.

s/Gateway/Main/

s/the Firebase backend/Firebase/